rest/app.py

```diff
@@ -71,16 +74,27 @@ def get_task(task_id):
 
 @app.route('/tasks', methods=['POST'])
 def add_task():
-    tasks.append(TASK_TO_ADD)
-    return json.dumps(tasks)
+    task = json.loads(request.data)
+    tasks.append(task)
+    return render_template('index.html'), 201
 
 
 @app.route('/tasks/<int:task_id>', methods=['DELETE'])
 def delete_task(task_id):
     for task in tasks:
         if task['id'] == task_id:
             tasks.remove(task)
-            return json.dumps(users)
+            return json.dumps(tasks)

```

Comment:
It's not a good idea, return tasks after removal, 
Please jsut send 200 or 204 - https://www.restapitutorial.com/httpstatuscodes.html